src/test/java/marquez/NamespaceBaseTest.java

```diff
@@ -0,0 +1,33 @@
+package marquez;
+
+import static java.time.Instant.now;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+import marquez.api.CreateNamespaceRequest;
+import marquez.core.models.Namespace;
+import marquez.dao.fixtures.AppWithPostgresRule;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+
+public class NamespaceBaseTest {
```

Comment:
Seems like just boilerplate stuff you're abstracting away? Unless it's commonly shared across tests, I'd just have this defined in `NamespaceIntegrationTest`.

Reply:
So there is some reuse in `NamespaceResourceTest`, but that also means `AppWithPostgresRule` is defined without ever being used, which does add to the overall run of the tests for `NamespaceResource` given we are setting up postgres.

Reply:
Good catch. Moved the app rule down to the class that needs it. Most everything else is used in both.